Cargo.toml

```diff
@@ -66,6 +72,13 @@ predicates = "3.0"
 pretty_assertions = "1.4"
 ring = "0.16.20"
 hex = "0.4.3"
+similar-asserts = "1.5.0"
+serde_json = { version = "1.0.107" }
+paste = "1.0.14"
+
+[[test]]
+name = "validation"
+required-features = ["validation"]
```

Comment:
Are they integration tests? They look like unit tests to me (except, the fs io)

Reply:
I mostly figured that they are, because
- They touch the filesystem
- They use data in the `tests/` directory

Want me to move them back?

Reply:
I think i would prefer them to be inlined (the local registry tests use the test/data dir aswell!) and use `include_str!` to get compile-time warnings if the test data is removed

Reply:
Sure!

I do prefer using `include_str!()` where possible, but unfortunately due to the way the `protobuf-parse` crate works, we always have to go by file name. So what I will do is use `include_str` for the expected JSON output, but still read the protobuf files from disk at runtime.